pages/project-version/pdf/views/nts/components/duration.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Duration({ version }) {
+  const duration = version.duration;
+  if (!duration) {
+    return;
+  }
+  let yearsLabel = duration.years === 1 ? 'Year' : 'Years';
+  let monthsLabel = duration.months === 1 ? 'Month' : 'Months';
+  return <p>{`${duration.years} ${yearsLabel} ${duration.months} ${monthsLabel}`}</p>;
+}
```

Comment:
do we not have standard renderers for fields like this in projects?

Reply:
There's this but it's a DL:
https://github.com/UKHomeOffice/asl-projects/blob/master/client/components/review-field.js#L37-L59

Reply:
and there's https://github.com/UKHomeOffice/asl-projects/blob/master/client/components/duration.js but I think that's an input?

Reply:
I would argue against displaying the same thing in a different way here - it's always a DL in granted, readonly, and PDF